3887 remove api key docs and mentions (#3904)
astronomer/docs: astro/cli/astro-workspace-create.md

| Option            | Description                                                                                                                             | Valid Values  |
| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
| `--description`   | The description for the Workspace.                                                                                                      | Any string    |

> | `--enforce-ci-cd` | Determines whether users are required to use a Workspace API token or Deployment API token to deploy code.  | `ON` or `OFF` |

Author: ```suggestion
| `--enforce-ci-cd` | Determines whether users are required to use an API token to deploy code.  | `ON` or `OFF` |
```
